Building error in Angular 19 projects

Hi,

When using the last version (2.19.6) with Angular 19 and the Vite bundler, I get this error:
```sh
✘ [ERROR] TS2309: An export assignment cannot be used in a module with other exported elements. [plugin angular-compiler]

    node_modules/proj4/dist/index.d.ts:5:0:
      5 │ export = proj4;
        ╵ ~~~~~~~~~~~~~~~
```
Can you fix it in a later version please?

Thanks a lot

--- a/index.d.ts
+++ b/index.d.ts
@@ -2,5 +2,5 @@ import proj4 from "./lib/index";
 export type { ProjectionDefinition } from "./lib/defs";
 export type { Converter, InterfaceCoordinates, TemplateCoordinates } from "./lib/core";
 export type { DatumDefinition } from "./lib/Proj";
-export = proj4;
+export default proj4;
 export as namespace proj4;

